opendevin/runtime/plugins/jupyter/setup.sh

```diff
@@ -2,14 +2,25 @@
 
 set -e
 
+# Hardcoded to use the Python interpreter from the OpenDevin runtime client
+OPENDEVIN_PYTHON_INTERPRETER=/opendevin/miniforge3/bin/python
+# check if OPENDEVIN_PYTHON_INTERPRETER exists and it is usable
+if [ -z "$OPENDEVIN_PYTHON_INTERPRETER" ] ||  [ ! -x "$OPENDEVIN_PYTHON_INTERPRETER" ]; then
+    echo "OPENDEVIN_PYTHON_INTERPRETER is not usable. Please pull the latest Docker image!"
+    exit 1
+fi
+
+# use mamba to install c library
+/opendevin/miniforge3/bin/mamba install -y gcc
```

Comment:
I'm thinking about whether we should pin the version of packages or not -- maybe something like `poetry.lock` we are using would be nice - We can do this in subsequent PR